tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/SqlInjection/EFCoreSqliteTests.cs

```diff
@@ -2,12 +2,14 @@
 
 using Microsoft.Data.Sqlite;
 using Microsoft.EntityFrameworkCore;
+using Xunit;
 
 namespace Samples.InstrumentedTests.Iast.Vulnerabilities.SqlInjection;
 
 #if !NET6_0_OR_GREATER
 [Xunit.Trait("Category", "AlpineArmUnsupported")] // sqlite isn't supported in .NET 5 on Alpine
 #endif
+[Collection("Non-Parallel Collection")]
```

Comment:
Note that adding these tests to a collection means they won't run in parallel with _anything_, which technically will slightly increase testing times... given we _only_ need to stop them running in parallel with each other (in theory) the other option would be to move the tests to the same type. But given the current structure that looks like it would be _much_ harder to do😅